src/topology/vector_bundle.lean

```diff
@@ -0,0 +1,106 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Nicolò Cavalleri.
+-/
+
+import topology.topological_fiber_bundle
+import analysis.calculus.deriv
+import linear_algebra.dual
+
+/-!
+# Topological vector bundles
+
+In this file we define topological vector bundles.
+
+I believe it is an enough important concept that it should deserve to be a class, unlike fiber
+bundles. The most important idea here is that vector bundles are named through their fibers.
+Let `B` be the base space. The collection of the fibers is a function `E : B → Type*` for which
+there is an appropriate instance on each `E x` and an instance of topological space over `Σ x, E x`.
+Naming conventions are essential to work with vector bundles this way.
+-/
+
+notation V `ᵛ` R := module.dual R V
```

Comment:
This should not occur in the final version of this file. Feel free to make this `localized` notation in `linear_algebra.dual` though.

Reply:
Ok sure

Reply:
How can the notation `V'` work? How can Lean guess the field?